paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -174,11 +172,7 @@ internal class CustomerSheetViewModel @Inject constructor(
 
         viewModelScope.launch {
```

Comment:
Is there any issue with us starting to observe this throughout the lifecycle of the view model instead of just when we enter the add screen?